base_projects/Drivetrain/src/main/include/logging/SysIdDrivetrainLogger.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include "logging/SysIdLogger.h"
+
+// TODO Set to proper number after telemetry data refactor
+class SysIdDrivetrainLogger : public SysIdLogger {
+ public:
+  /**
+   * The users should their left motors to what this returns AFTER calling log.
+   */
+  units::volt_t GetLeftMotorVoltage();
+
+  /**
+   * The users should set their right motors to what this returns AFTER calling
+   * log.
+   */
+  units::volt_t GetRightMotorVoltage();
+
+  /**
+   * Logs data for a drivetrain mechanism.
+   * Outputs data in form: timestamp, l voltage, r voltage, l position, r
+   * position, l velocity, r velocity, angle, angular rate
```

Comment:
This should be more like
```suggestion
   * Logs data for a drivetrain mechanism.
   *
   * Outputs data in form: timestamp, l voltage, r voltage, l position, r
   * position, l velocity, r velocity, angle, angular rate.
   *
   * @param leftPosition Something descriptive.
   * @param rightPosition Something descriptive.
   * <you get the idea>
```